### PR TITLE
Update images for Monitoring rebase to 11.0.0

### PR DESCRIPTION
**Pull request by aiyengar2**
These image updates are required for https://github.com/rancher/charts/pull/836.

#### Pull Request Checklist ####

- [x] Change does not remove any existing Images or Tags
- [x] New entries are in format `SOURCE DESTINATION TAG`
- [x] New entries are added to the correct section of the list (sorted lexographically) 
- [x] Changes to scripting or CI config have been tested to the best of your ability

#### Types of Change ####

New image tags for:
- curlimages/curl rancher/curlimages-curl 7.73.0
- grafana/grafana rancher/grafana-grafana 7.2.1
- jettech/kube-webhook-certgen rancher/jettech-kube-webhook-certgen v1.5.0
- jimmidyson/configmap-reload rancher/jimmidyson-configmap-reload v0.4.0
- kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 1.1.0

New images that require new repositories:
- grafana/grafana-image-renderer rancher/grafana-grafana-image-renderer 2.0.0
    - Created a ticket to create the new repository

New images that have existing Rancher repositories but have been added to the scripts for the first time:
- quay.io/prometheus-operator/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.43.0
    - https://hub.docker.com/r/rancher/coreos-prometheus-config-reloader/tags
- quay.io/prometheus-operator/prometheus-operator rancher/coreos-prometheus-operator v0.43.0
    - https://hub.docker.com/r/rancher/coreos-prometheus-operator/tags
- quay.io/prometheus/prometheus rancher/prom-prometheus v2.22.0
    - https://hub.docker.com/r/rancher/prom-prometheus/tags

#### Linked Issues ####

PR: https://github.com/rancher/charts/pull/836

Related Issue: https://github.com/rancher/rancher/issues/29871

### CODE REVIEW

File: images-list [hunk 1]
```diff
@@ -51,6 +57,9 @@ prom/prometheus rancher/prom-prometheus v2.18.2
 quay.io/kiali/kiali rancher/kiali-kiali v1.22.1
 quay.io/kiali/kiali rancher/kiali-kiali v1.23.0
 quay.io/kiali/kiali rancher/kiali-kiali v1.24.0
+quay.io/prometheus-operator/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.43.0
+quay.io/prometheus-operator/prometheus-operator rancher/coreos-prometheus-operator v0.43.0
+quay.io/prometheus/prometheus rancher/prom-prometheus v2.22.0
```
Comment (aiyengar2) [inline images-list:62]: @cbron as mentioned in the issue description, the following already have Rancher repos but aren't in the scripts:
- https://hub.docker.com/r/rancher/coreos-prometheus-config-reloader/tags
- https://hub.docker.com/r/rancher/coreos-prometheus-operator/tags
- https://hub.docker.com/r/rancher/prom-prometheus/tags

Do we need an EIO ticket to track enabling the repositories?

Comment (cbron) [reply images-list:62]: They shouldn't, but we can ask EIO to double check that they have drone -> docker credentials set correctly.

**Review by cbron — COMMENTED** [review]
lgtm but waiting to approve until new hub repo is created

File: images-list (hunk 1 above)
Comment (aiyengar2) [reply images-list:62]: They verified the permissions, so it should be good to go.

**Comment by aiyengar2** [comment]
> lgtm but waiting to approve until new hub repo is created

Exists at https://hub.docker.com/r/rancher/grafana-grafana-image-renderer/tags

**Review by cbron — APPROVED** [review]

**Pull request merged**